### DISCUSSION TITLE
October 17th 2024 Community Call

### DISCUSSION THREAD

**Discussion by SydneyhSmith** [PowerShell/PowerShell#24362] [Community Call]
Agenda: 

- November Call (celebration/last of the year) (Sydney)
- 7.2 EOL  (Sydney)
- OpenSSH 9.8 release (Tess)
- VSCode Extension and PSSA updates (Andy)
- Docs update (Sean)
- Azure MFA (Michael)
- Issue https://github.com/PowerShell/PowerShell/discussions/24340 (Sydney)
- @kilasuit demo

Please post topics/questions/demos.

We will accept demos posted 24 hours in advance-- thanks 😄

**Comment by kilasuit** [comment]
I committed last month in demoing the output from me rasing [Add a mechanism to download a build artifact - 24081](#24081)

**Reply by SydneyhSmith** [reply]
Awesome!

**Comment by floh96** [comment]
Please talk about https://github.com/PowerShell/PowerShell/discussions/24340 I'm curious, last time I checked the standpoint of the pwsh team was that it is not possible to ship pwsh 7 with windows because of the different support cycles from windows and .NET.

**Reply by iSazonov** [reply]
> it is not possible to ship pwsh 7 with windows because of the different support cycles from windows and .NET.

Yes, it's weird. 

Will the support cycle be changed for Windows or for pwsh? Will .NET be inbox on Windows? As for old Windows versions?

**Reply by SydneyhSmith** [reply]
Yep-- can definitely add this topic to the agenda
